src/bosh_openstack_cpi/Gemfile

```diff
@@ -1,11 +1,15 @@
 source 'https://rubygems.org'
 
-gemspec
+gem 'bosh_common'
+gem 'bosh_cpi'
+gem 'fog-openstack'
```

Comment:
We should probably lock down the version of fog-openstack to something specific here. This will horribly break us if something goes wrong.